platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
Sure :) The only reason I chose `protected` is to have default base behavior (eg. format date and changing the case for the log level), so that it's not smeared across all of the appenders.  Do you want all appenders to do argument formatting independently?